packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -637,7 +637,18 @@ export class ProjectService {
                           rows: rows.length,
                           warehouse: warehouseConnection?.type,
                       },
-                      async () => formatRowsWorker(rows, itemMap),
```

Comment:
should we delete the `formatRowsWorker` function from this file ? I don't think it is being used anymore.

Reply:
oh yeah, good catch